requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
```

Comment:
@pepol a very small nitpick, could you please lock this with the package version? It's not a big deal but there are quite a few versions of flask out there and I'd rather be safe now than sorry afterwards.